### PR TITLE
Portfolio: details popup window

### PR DESCRIPTION
**Pull request by jovasal**
<h1>Portfolio: details popup window</h1>

1️⃣ You should implement the following interactions:

- When the user clicks (or taps) the button to check project details, the popup with details about the project appears.
- When the user clicks (or taps) the close (X) button, the popup disappears.

2️⃣ In order to associate each project with the popup window details, you must refactor the project section:

- You need to use a JavaScript array to store all of the information for all projects.
- For each project you need to store the following pieces of data in a JavaScript object, at a minimum: name, description, featured image, technologies, link to live version, link to source.
- You need to update the main page so that the projects section is created dynamically using the information stored in that JavaScript object. Remember, all of the HTML in that section is created when the page loads.
- After that, you could implement the popup window.

3️⃣ You should implement popups for both mobile and desktop screen sizes:

-Mobile popup

- Desktop popup

4️⃣ You must stick to the design as much as possible (e.g., font, colors, images, text, margins) using figma

### CODE REVIEW

**Review by 1GORDON — COMMENTED** [review]
##
## Hi @jovasal. and @Nessrine88 ✋

Good job so far!

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

### This project will be marked as Invalid and won't affect the number of your weekly reviews.

- [ ] Kindly remove the live demo from your readme file since this project is not yet deployed.

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: script.js [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+  popupContainer.style.display = 'none';
+});
+
+button1.addEventListener('click', () => {
+
+});
+
+button2.addEventListener('click', () => {
+
+});
```
Comment (Strangeal) [inline script.js]: - [x] Great job on implementing the hamburger menu as specified in the project requirement! 👏 However, there is a minor UI issue 🐞 that we need to address and fix to help create a good user experience. 
   - [x] Kindly consider removing these `event` listeners from your codebase as they are not serving any purpose. Having unused code adds unnecessary complexity and potential performance overhead. Removing these unused event listeners will help improve code clarity and maintainability.

**Review by Strangeal — CHANGES_REQUESTED** [review]
Hi @jovasal, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

Alt Text
## Highlights
Correct use of GitHub Flow ✅
Well-structured files ✅
All linter checks are passing ✅
Descriptive PR title with a great summary ✅

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: script.js (hunk 1 above)
Comment (jovasal) [reply script.js]: Thank you so much!!

I removed:

button1.addEventListener('click', () => {

});

button2.addEventListener('click', () => {

});

Code is working.

File: index.html [hunk 2]
```diff
@@ -1,244 +1,465 @@
...
+              sign-ups required.
+            </p>
+            <ul class="stacks">
+              <li class="stack-over">HTML</li>
+              <li class="stack-over">CSS</li>
+              <li class="stack-over">Javascript</li>
+            </ul>
+            <!-- <a href="#" class="btn">See Project</a> -->
+            <button class="btn" id="popupButton1">See Project</button>
+          </div>
```
Comment (frempongdev) [inline index.html:125]: Great job so far implementing the popup. 👍🏿  There are some few changes I would suggest to complete this project. ♻️ 

[ OPTIONAL ]
- [ ] Kindly remove the sections block from your HTML file as it is required for you to generate it dynamically as the page loads. 🗑️ 

[ OPTIONAL ]
- [ ] Kindly ensure that your popup is the only active part of your page when in view as it is a good User experience practice. Currently you main page seems to have an active scroll when your popup is in view. 
aboutgsg


[ OPTIONAL ]
- [ ] Kindly make sure that your design looks similar to the given Figma template as that is the core of this section of your learning. Paying attention to details can go a long way to help you satisfy your future clients. 🧰 

### Your design ⬇️
image

### Figma design ⬇️  
image

**Review by frempongdev — APPROVED** [review]
# STATUS: APPROVED 🟢 ✔️  

Hi @jovasal , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
Professional Readme :heavy_check_mark: 
all application features work :heavy_check_mark:

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request closed without merging**